pygbm/histogram.py

```diff
@@ -20,6 +20,40 @@ def _build_ghc_histogram_naive(n_bins, sample_indices, binned_feature,
     return histogram
 
 
+@njit
+def _build_ghc_histogram_unrolled_fast(n_bins, parent_histogram,
+                                       sibling_histogram):
```

Comment:
I would rename this a `_substract_histograms` and `parent_histogram` to `hist_a` and `sibling_histogram` to `hist_b`.

Reply:
Also, I think that you should create a new method name `find_node_split_by_substraction` that would call a new function named `_parallel_substract_histograms`.

And keep `find_node_split` for the case where we actually compute histograms with `_parallel_find_splits`.

Reply:
Yes this definitely needs some refactoring. Thanks for all the comments, I'll go back to it soon.

Also, any idea on how to report execution time from inside numba? importing the `time` module does not work when in no-python mode. We're currently reporting time(hist + splittability) and what we're interested in is only time(hist).